bouncer/templates/annotation.html.jinja2

```diff
@@ -1,17 +1,17 @@
 {% extends "templates/base.html.jinja2" %}
 
-{% set title = _("Hypothesis: Redirecting you&hellip;") %}
+{% set title = _("Hypothesis: Annotation for " ) + pretty_url  %}
 
 {% block content %}
   <div class="center spinner__stationary-ring">
-    <img alt="Redirecting"
+    <img alt="Loading annotation for {{ pretty_url }}"
           class="center spinner__icon"
           height="24"
           src="{{'bouncer:static/images/hypothesis-icon.svg'|static_path}}"
           width="28">
     <div class="spinner__moving-ring"></div>
   </div>
-  <p class="spinner__text">{% trans %}Redirecting{% endtrans %}</p>
+  <p class="spinner__text">{% trans %}Loading annotation for {% endtrans %}{{ pretty_url }}</p>
```

Comment:
Again `pretty_url` needs to be part of the string that gets passed to the translator so they can move it around in the sentence if necessary. On this line we're using the `{% trans %}` tag so I think you can actually just do this:

```jinja2
{% trans %}Loading annotation for {{ pretty_url }}{% endtrans %}
```